### PR TITLE
Pods 7045

### PR DESCRIPTION
**Pull request by sqinnydave**

### CODE REVIEW

File: app/assets/stylesheets/pensionsschemefrontend-app.scss [hunk 1]
```diff
@@ -10,9 +10,9 @@
 @import 'partials/_shame';
 
 @import 'partials/page-heading';
-@import 'partials/scheme__list';
+// @import 'partials/scheme__list';
 
-@import 'location-autocomplete.min';
+// @import 'location-autocomplete.min';
```
Comment (ngmjohnstone) [inline app/assets/stylesheets/pensionsschemefrontend-app.scss:16]: Wondering what the case is for leaving these commented lines in?

Comment (sqinnydave) [reply app/assets/stylesheets/pensionsschemefrontend-app.scss:16]: when we do the cleanup ticket there's a task to remove the previous assets (js and css) and to only link the new equivalents. to this end the autocomplete css had been moved into the ...twirlupgrade.css file. the 2 views that use autocomplete (established country and address component) have been updated to new fe and merged to 6212 so no pages require that css anymore so we commented it out as a placeholder that this wont concern us. But as the file will be removed at the end it's a moot point. as is this whole explanation really :) but i just wanted to give you some background on the rationale.

File: app/views/address/addressList.scala.html [hunk 2]
```diff
@@ -15,55 +15,64 @@
...
 @import utils.ViewHelper.titleWithErrors
 
 @this(
-    main_template: main_template,
-    formHelper: FormWithCSRF
+    layout: templates.Layout,
+    formHelper: PlayFrontendFormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios : GovukRadios,
+    input_radio: components_refactored.input_radio
```
Comment (jigneshkumarpatel) [inline app/views/address/addressList.scala.html:31]: input_radio we can remove as can see we are using govukRadios only

**Review by jigneshkumarpatel — CHANGES_REQUESTED** [review]

File: app/views/address/addressList.scala.html (hunk 2 above)
Comment (sqinnydave) [reply app/views/address/addressList.scala.html:31]: will resolve this with a push

Comment (sqinnydave) [reply app/views/address/addressList.scala.html]: this is done now @jigneshkumarpatel

**Review by jigneshkumarpatel — APPROVED** [review]

**Pull request merged**